### ISSUE TITLE
CPC loss

### ISSUE THREAD

**Issue by Liujingxiu23** [bshall/VectorQuantizedCPC#9]
I read related papers， but still do not understand the CPC loss computaiton.

            labels = torch.zeros(
                self.n_speakers_per_batch * self.n_utterances_per_speaker, length,
                dtype=torch.long, device=z.device
            )

            loss = F.cross_entropy(f, labels)

Can someone explain it for me. Why labels of zeros and cross_entropy used here?

**Comment by Liujingxiu23** [comment]
Get explaination in VQMIVC.
